### DISCUSSION TITLE
How to get rid of the hover-over visual style in the ListView

### DISCUSSION THREAD

**Discussion by HakanL** [unoplatform/uno#12795] [Q&A] [answered]
I have a list of items in a ListView and I have SelectionMode="None", the user isn't supposed to select anything, it's just a list of items (that is scrollable). But I want to prevent the hover-over visual style that shows as a dark gray background, how do I go about that? I've tried to override the Focused VisualState, but it doesn't seem to be it. I can set IsEnabled, but then all my list items are grayed out.
![image](https://github.com/unoplatform/uno/assets/407941/5cdc1d0e-c19c-4dc3-96d3-b20e4f5f8677)

**Comment by jeromelaban** [comment]
If you're trying to get a list of items without the listview features like selection, but with virtualization, you may want to take a look at the [`ItemsRepeater` control](https://learn.microsoft.com/en-us/windows/apps/design/controls/items-repeater).

**Reply by HakanL** [reply]
Yeah, I think I realized a little too late that the ItemsRepeater would be better. I may look into switching it out. I guess I was approaching it from the angle of making the ListView fit my needs.

**Reply by jeromelaban** [reply, marked as answer]
It's possible to adjust the ListView as well, by redefining the style of a `ListViewItem`, though `ItemsRepeater` can be more configurable.
